manifests/autotune/performance-profiles/resource_optimization_local_monitoring.yaml

```diff
@@ -246,3 +246,168 @@ slo:
     aggregation_functions:
     - function: max
       query: 'max by(namespace,container) (last_over_time((timestamp(container_cpu_usage_seconds_total{namespace="$NAMESPACE$", container="$CONTAINER_NAME$"} > 0))[15d:]))'
+
+    ## namespace related queries
+
+    # Namespace quota for CPU requests
+    # Show namespace quota for CPU requests in cores for a namespace
+    - name: namespaceCpuRequest
+      datasource: prometheus
+      value_type: "double"
+      kubernetes_object: "namespace"
+      aggregation_functions:
+        # sum of all cpu request quotas for a namespace in cores
+        - function: sum
```

Comment:
Please follow same spacing as in the above cases to maintain consistency. 

```

# Incorrect
aggregation_functions:
   # sum of all cpu request quotas for a namespace in cores
   - function: sum
 
# Correct
aggregation_functions:
# sum of all cpu request quotas for a namespace in cores
- function: sum        
```

Reply:
I have corrected the spacing to maintain consistency. Could you please review it again and let me know your comments?